debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -904,7 +915,7 @@ protected String slotName() {
     }
 
     protected boolean dropSlotOnStop() {
-        return getConfig().getBoolean(DROP_SLOT_ON_STOP);
+        return getConfig().hasKey(DROP_SLOT_ON_STOP_OBSOLETE.name()) ? getConfig().getBoolean(DROP_SLOT_ON_STOP_OBSOLETE) : getConfig().getBoolean(DROP_SLOT_ON_STOP);
```

Comment:
If both are given, old and new option, the latter should take precedence.

Reply:
@gunnarmorling Good point! Added